third_party/lint/repos.bzl

```diff
@@ -17,19 +17,15 @@ def lint_repos(lowrisc_lint = None):
     # that golang depends on.
     http_archive(
         name = "ot_hack_bazelbuild_buildtools",
```

Comment:
If we no longer need to patch this, can we rename the repo to not be `ot_hack_...`?